### DISCUSSION TITLE
Will this Arduinocore work with the Apollo3 Plus? (2MB flash version)

### DISCUSSION THREAD

**Discussion by curtpw** [sparkfun/Arduino_Apollo3#479] [Q&A]
Will this arduinocore/bootloader work with the Apollo3 Plus? ie https://ambiq.com/apollo3-blue-plus/

What would I need to get this working on the Plus version? FYI I'm hacking a cheap smartwatch 
![Apollo3BluePlus_Differences](https://user-images.githubusercontent.com/3660857/215435457-a9fc66fa-0992-4927-a7cd-f7a9816da8ce.jpg)

**Comment by JaredSapphire** [comment]
Here are some forks for the Apollo3 Blue Plus that may help.
https://github.com/SapphireCircuits/Arduino_Apollo3Plus
https://github.com/SapphireCircuits/Apollo3Plus_Uploader_SVL
